chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -587,13 +587,34 @@
         mode 0644
         owner "root"
         group "root"
-        source "crowbar_join.ubuntu.sh.erb"
-        variables(admin_web: admin_web,
-                  os_codename: os_codename,
-                  crowbar_repo_web: crowbar_repo_web,
-                  admin_ip: admin_ip,
-                  provisioner_web: provisioner_web,
-                  web_path: web_path)
+        source "crowbar_join.sh.erb"
+        variables(admin_ip: admin_ip,
+                  web_port: web_port,
+                  ntp_servers_ips: ntp_servers_ips,
+                  platform_family: "debian",
+                  target_platform_version: target_platform_version)
+      end
+
+      if os == "debian-8"
+        param_install_url = params["install_url"]
+        debian_arch = case arch
+          when "x86_64"
```

Comment:
Style/CaseIndentation: Indent `when` as deep as `case`. (https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)